clients/native/src/client/mod.rs

```diff
@@ -176,9 +181,19 @@ impl NymClient {
         .await
         .expect("could not obtain aggregate verification key of validators");
 
+        let params = Parameters::new(TOTAL_ATTRIBUTES).unwrap();
+        let bandwidth_credential_attributes = BandwidthVoucherAttributes {
+            serial_number: params.random_scalar(),
+            binding_number: params.random_scalar(),
+            voucher_value: hash_to_scalar(BANDWIDTH_VALUE.to_be_bytes()),
+            voucher_info: hash_to_scalar(String::from("BandwidthVoucher").as_bytes()),
```

Comment:
Regarding checking crypto I think you don't have to.
Regarding the 'BandwidthVoucher', this is a temporary thing until we figure out what we actually want to keep as the (public) voucher info. This can be a voucher type, or an epoch number or something else.